mailbox/backup/src/main/java/org/apache/james/mailbox/backup/Backup.java

```diff
@@ -35,5 +32,5 @@
      * @param messages a stream of MailboxMessages that will be consumed
      * @param destination an OutputStream in which the zip will be written
      */
-    public void archive(Map<Mailbox, List<MailboxAnnotation>> mailboxes, Stream<MailboxMessage> messages, OutputStream destination) throws IOException ;
+    public void archive(List<MailboxWithAnnotations> mailboxes, Stream<MailboxMessage> messages, OutputStream destination) throws IOException ;
```

Comment:
please also change the parameter name

Reply:
I think the name is fine, but i should update the comments to reflect the fact that the mailboxes come with their annotations.